### PR TITLE
Add magnifier widget options sample example using view.xml

### PR DESCRIPTION
**Pull request by ParthBrahmbhatt**
## Purpose of this PR

This PR improves the magnifier widget documentation with some useful content which can be used on widget's initialization with different parameter using view.xml file.

## Affected URLs

- v2.3 https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_gallery_mg.html
- v2.2 https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_gallery_mg.html
- v2.1 https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_gallery_mg.html

## Links to source code

- https://github.com/magento/magento2/blob/2.1/app/design/frontend/Magento/blank/etc/view.xml#L213

whatsnew
Added an example of configuring the initialization of the [magnifier widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_gallery_mg.html) with different parameters using the `view.xml` file.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/javascript-dev-guide/widgets/widget_gallery_mg.md [hunk 1]
```diff
@@ -98,3 +98,19 @@ Possible values:
...
+
+```xml
+<var name="magnifier">
+    <var name="%magnifier_option1%">%option1_value%</var>
+    <var name="%magnifier_option2%">%option2_value%</var>
+...
+</var>
+```
+
+For illustration of setting gallery option in `view.xml`, you can reference to the [view.xml of the Blank theme]({{ site.mage2000url }}app/design/frontend/Magento/blank/etc/view.xml#L225).
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_gallery_mg.md]:
```suggestion
For an example of setting the gallery option, see the [view.xml]({{ site.mage2000url }}app/design/frontend/Magento/blank/etc/view.xml#L225) file in the Blank theme.
```

File: guides/v2.1/javascript-dev-guide/widgets/widget_gallery_mg.md [hunk 2]
```diff
@@ -98,3 +98,19 @@ Possible values:
 
 * `outside`
 * `inside`
+
+## Configure magnifier options in `view.xml`
+
+Magnifier options can be set in the `view.xml` configuration file of a theme. The file is conventionally located in `<theme_dir>/etc`.
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_gallery_mg.md]:
```suggestion
Magnifier options can be set in the `view.xml` configuration file of a theme. The file is conventionally located in the `<theme_dir>/etc/` directory.
```

**Comment by ParthBrahmbhatt** [comment]
Hello @jeff-matthews and @ronak2ram 
I have updated the content as per the suggetion.
Thanks

File: guides/v2.1/javascript-dev-guide/widgets/widget_gallery_mg.md [hunk 3]
```diff
@@ -98,3 +98,19 @@ Possible values:
...
+
+```xml
+<var name="magnifier">
+    <var name="%magnifier_option1%">%option1_value%</var>
+    <var name="%magnifier_option2%">%option2_value%</var>
+...
+</var>
+```
+
+For an example of setting the gallery option, see the [view.xml]({{ site.mage2000url }}app/design/frontend/Magento/blank/etc/view.xml#L225) file in the Blank theme.
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_gallery_mg.md]:
```suggestion
For an example of setting the gallery option, see the [view.xml]({{ site.mage2100url }}app/design/frontend/Magento/blank/etc/view.xml#L225) file in the Blank theme.
```

**Comment by ParthBrahmbhatt** [comment]
Hi @jeff-matthews 
I have updated the latest changes given by you.

**Pull request merged**